src/baguetteBox.js

```diff
@@ -313,8 +327,19 @@
         if (options.buttons === 'auto' && ('ontouchstart' in window || imagesMap[currentGallery].length === 1)) {
             options.buttons = false;
         }
-        // Set buttons style to hide or display them
-        previousButton.style.display = nextButton.style.display = (options.buttons ? '' : 'none');
+
+        if (options.buttons === false) {
+            previousButtonContent.setAttribute('aria-hidden', 'true');
+            nextButtonContent.setAttribute('aria-hidden', 'true');
+
+            // Set buttons style to hide
+            previousButton.style.display = nextButton.style.display = 'none';
+        } else {
+            previousButtonContent.setAttribute('aria-hidden', 'false');
+            nextButtonContent.setAttribute('aria-hidden', 'false');
+            closeButtonContent.setAttribute('aria-hidden', 'false');
+        }
```

Comment:
You mean I don't need to add `aria-hidden=true` around the SVG images?

I was under the impression we always needed to do that but all this accessibility stuff are quite new to me.